.github/workflows/test.yml

```diff
@@ -109,55 +116,59 @@ jobs:
         with:
           toolchain: ${{ matrix.config.rust-version }}
           components: rustfmt, clippy
+          targets: ${{ matrix.config.target }}
 
-      - name: Configure targets
+      # All configurations for Windows go here
+      # 1. Configure linker if RTools >= 4.2
+      # 2. Determine mingw root
+      # 3. Create libgcc_eh mock for RTools >= 4.2
+      # 4. Add R bin path to PATH
+      # 5. Add mingw root/bin to PATH (required to find linker)
+      # 6. Add include path (required to resolve standard headers like stdio.h)
+      - name: Configure Windows
+        if: runner.os == 'Windows'
         run: |
-          if ($env:RUST_TARGETS -eq '') {
-            $env:RUST_TARGETS = "default"
-          }
-          foreach ($target in ($env:RUST_TARGETS).Split(",")) {
-            if ($target -ne "default") {
-              rustup target add $target
+          choco install llvm -y --version 16.0.1 --force
```

Comment:
Sorry to comment here again. Why do we need a specific version of LLVM? LLVM is a bit heavy to install (lighter than installing R, though), so I think it's nice if we can skip this.

Reply:
Ok, but we will be using 'older' version of LLVM compared to MSYS2